app/controllers/v1/rule_results_controller.rb

```diff
@@ -4,11 +4,15 @@ module V1
   # API for RuleResults
   class RuleResultsController < ApplicationController
     def index
-      render json: RuleResultSerializer.new(scope_search, metadata)
+      render_json scope_search
```

Comment:
`render_json` in rendering.rb gives automatic side loaded includes (the `include` parameter), metadata, and error handling.